html_text/html_text.py

```diff
@@ -39,7 +39,13 @@ def _cleaned_html_tree(html):
         tree = html
     else:
         tree = parse_html(html)
-    return cleaner.clean_html(tree)
+
+    try:
+        cleaned = cleaner.clean_html(tree)
+    except Exception:
+        cleaned = lxml.html.Element('html')
```

Comment:
This is essentially an empty element, right? Another option would be to return ``tree`` unchanged - that would be less surprising for me, probably. Do you think returning an empty result is better in this base?